examples/buildpacks/buildpacks.cue

```diff
@@ -23,7 +23,7 @@ frsca: pipelineRun: "cache-image-pipelinerun-": spec: {
 		value: "latest"
 	}, {
 		name:  "SOURCE_URL"
-		value: "https://github.com/buildsec/example-buildpacks"
+		value: "https://gitea-http.gitea:3000/frsca/example-buildpacks"
```

Comment:
Agreed, we've got a lot of that to clean up across all the examples. It does need to be a default though, since examples like cosign didn't make sense to ingest.

Reply:
Agreed, the value in examples could be something like the following:
```
examples: repo: string | *"https://gitea-http.gitea:3000/frsca"
```
Then the usage here could be something like:
```
value: "\(examples.repo)/example-buildpacks"
```

Reply:
I've updated with:

```
_GIT_ORG: *"https://gitea-http.gitea:3000/frsca" | string @tag(gitOrg)
```

same way we do repository and image today. Moving all of that into an example data structure would be interesting for a future PR.